<ui version="4.0" >
 <class>SMPPopup</class>
 <widget class="QWidget" name="SMPPopup" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>111</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="1" colspan="3" >
    <widget class="QLabel" name="lMessage" >
     <property name="text" >
      <string>Please enter the secret passphrase to authenticate this contact.</string>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="3" >
    <widget class="KLineEdit" name="lePassphrase" />
   </item>
   <item row="2" column="0" >
    <widget class="KPushButton" name="pbHelp" >
     <property name="text" >
      <string>&amp;Help</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="KPushButton" name="pbManualAuth" >
     <property name="text" >
      <string>&amp;Manual Authentication</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" >
    <widget class="KPushButton" name="pbCancel" >
     <property name="text" >
      <string>&amp;Cancel</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3" >
    <widget class="KPushButton" name="pbOk" >
     <property name="text" >
      <string>&amp;OK</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
